rb/lib/selenium/webdriver/common/driver.rb

```diff
@@ -174,14 +175,20 @@ def quit
       ensure
         @service_manager&.stop
         @devtools&.close
+        @bidi&.close
       end
 
       #
       # Close the current window, or the browser if no windows are left.
       #
 
       def close
-        bridge.close
+        open_window_handles = bridge.close
```

Comment:
If you rewrite this as.
```
bridge.close tap do |open_handles|
  @bidi&.close if open_handles&.empty?
end
```

It'll run slightly more optimally and be less LOC

Reply:
Return value is always an array (or it will error);
And Rubocop will want it with braces on one line:
```
bridge.close.tap { |handles| @bidi&.close if handles.empty? }
```

Reply:
Thank you @luke-hill and @titusfortner for the suggestion. The suggested change works.

Also, @titusfortner as you say "_Return value is always an array (or it will error)_", that approach makes a unit test fail, which is...
https://github.com/SeleniumHQ/selenium/blob/trunk/rb/spec/unit/selenium/webdriver/support/event_firing_spec.rb#L148

with the error
```
NoMethodError:
       undefined method `empty?' for nil:NilClass
```
That's why I think nil value can be returned. And hence I should use `handles&.empty?`. Let me know if there's a better approach.